.env.example

```diff
@@ -19,3 +19,6 @@ AWS_PINPOINT_REGION=us-west-2
 AWS_EMF_ENVIRONMENT=local
 
 CONTACT_FORM_EMAIL_ADDRESS = ""
+
+AWS_PINPOINT_POOL_ID=
+AWS_PINPOINT_TEMPLATE_IDS=
```

Comment:
Are these two would be meant for the short code specifically? Because on long term, we will want to have a several pools with a 1:M mapping to templates, we might want to specify for now this is only for short code in the environment name. It might be better too in case that this stays this way for longer than we expect and cannot complete the short code integration work as we plan it at the moment.